### ISSUE TITLE
Pop-up Turrets: Hidden Buildings

### ISSUE THREAD

**Issue by VampireSeraphin** [MegaMek/megamek#522]
Tried this a while ago to try and build some pop-up turrets. The elevation of the turrets and buildings do not change when the elevator activates.

**Comment by psikomonkie** [comment]
I'm not sure if buildings / turrets should move with elevators strictly, but I think this should definitely stay open and, if needed, be edited to be called "Support pop-up turrets".

**Comment by HammerGS** [comment]
As mentioned on the Discord, the universe is full of Pop up things. But the rules and fluff lean heavily toward them being hidden units

**Comment by psikomonkie** [comment]
Right! So in that case, I've renamed this, and marked this as outdated. If someone verifies that hidden buildings work (the building terrain generated during deployment will be visible, but to other players it should be be indistinguishable from other buildings until "un-hidden") then this should be closed.

**Comment by HammerGS** [comment]
Closing as part of the Gun Emplacement clean-up. The legacy Gun Emplacement unit type (a turret that rides on a map building) is being deprecated in favour of TO:AR Advanced Buildings, which are full units with their own CF, armor and hex. See #8291. This report is against the old unit type and a version several years out of date, so it will not be fixed there. If you see the same behaviour with an Advanced Building on a current nightly, please open a new issue with the Advanced Building label.
